apis/kueue/v1beta1/workload_types.go

```diff
@@ -125,6 +125,12 @@ type PodSetTopologyRequest struct {
 	// For example, in the context of JobSet this value is read from jobset.sigs.k8s.io/replicatedjob-replicas.
 	SubGroupCount *int32 `json:"subGroupCount,omitempty"`
 
+	// PodSetGroup indicates the name of the group of PodSets to which this PodSet belongs to.
+	// PodSets with the same `PodSetGroup` should be assigned the same ResourceFlavor
+	//
+	// +optional
+	PodSetGroup *string `json:"podSetGroup,omitempty"`
```

Comment:
```suggestion
	PodSetGroupName *string `json:"podSetGroupName,omitempty"`
```
To align with the KEP: https://github.com/kubernetes-sigs/kueue/tree/main/keps/2724-topology-aware-scheduling#internal-apis